src/components/AdminContainer.js

```diff
@@ -148,20 +150,19 @@ class AdminContainer extends Component {
     return (
       <React.Fragment>
         {adminAddress ? (
-          <Panel>
-            {baseStyle}
-            <div className="flex">
-              <div className="w-50">
-                <h1 className="mt0">Admin</h1>
-              </div>
-              <div className="w-50">
-                <NetworkSelectorContainer />
-              </div>
-            </div>
-            <div className="flex bb pb3">
-              <div className="w-50">
-                <h3>
-                  Current account{" "}
+          <React.Fragment>
+            <nav className="dt pa3 w-100 border-box pa3 ph5-ns bg-white shadow-1-ns flex-l flex-row-l">
+              <a className="w-100 w-20-l" href="/" title="Home">
+                <img
+                  src={"../../static/images/logo.svg"}
+                  className="dtc v-mid mid-gray link dim pa3"
+                  style={{ width: "200px" }}
+                  alt="OpenCerts"
+                />
+              </a>
+              <div className="w-100 w-40-l">
```

Comment:
Instead of using fixed ratio ie.40%. Can we leave the icon on the left and flush everything to the right? The space between current account and account balance is looking weird.

Reply:
Updated.